src/Explorer/Panes/PanelContainerComponent.tsx

```diff
@@ -77,11 +78,20 @@ export class PanelContainerComponent extends React.Component<PanelContainerProps
   }
 
   private onDissmiss = (ev?: KeyboardEvent | React.SyntheticEvent<HTMLElement>): void => {
+    const collection = useSelectedNode.getState().findSelectedCollection();
+    const targetElementDataTest: string = collection.id();
```

Comment:
Why this middle variable? Couldn't the next line be instead:
const targetElement: HTMLElement | null = document.querySelector(`[data-test="${collection.id()}"]`)